modules/lang-painless/src/main/java/org/elasticsearch/painless/FunctionRef.java

```diff
@@ -66,6 +66,9 @@
     /** delegate method type method as type */
     public final Type delegateType;
 
+    /** whether a call is made on a delegate interface */
+    public final int isDelegateInterface;
```

Comment:
I wonder if it'd be worth making this a `boolean` for readability. I'm fine with it staying an integer if you like but I think it is worth a comment why it is an integer and that it can only be 0 or 1.

Reply:
Fixed and commented.